### PR TITLE
Extensible HAL links

### PR DESCRIPTION
**Pull request by whiskeysierra**

### CODE REVIEW

File: hyper-media/Hypermedia.md [hunk 1]
```diff
@@ -52,6 +52,8 @@ We opted for this subset of HAL after conducting a comparison of different hyper
 | [Siren](https://github.com/kevinswiber/siren)                  | ✗          | ✗             | ✗        | Entities and navigation | ✗    |
 | [Collection+JSON](http://amundsen.com/media-types/collection/) | ✗          | ✗             | ✗        | Collections and queries | ✗    |
 
+We define HAL links to be extensible, i.e. to contain additional properties if needed. For consistency extensions should reuse attributes from the [`Link` header](https://tools.ietf.org/html/rfc5988#page-8).
```
Comment (tkrop) [inline hyper-media/Hypermedia.md]: @whiskeysierra shouldn't the link better refer to `https://tools.ietf.org/html/rfc5988#section-5`

Comment (fmueller) [inline hyper-media/Hypermedia.md]: +1 for changing the link to correct section in corresponding RFC.

Rest LGTM.

**Comment by whiskeysierra** [comment]
I updated the link

**Comment by fmueller** [comment]
👍

**Comment by whiskeysierra** [comment]
:+1:

**Pull request merged**